src/Goerli-DssSpellCollateral.sol

```diff
@@ -30,58 +68,180 @@ contract DssSpellCollateralAction {
     //
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
-    //
+
+    uint256 constant ZERO_PCT_RATE                  = 1000000000000000000000000000;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+    uint256 public constant RAY = 10**27;
+    uint256 public constant RAD = 10**45;
+
+    // -- RWA007 MIP21 components --
+    address constant RWA007                         = 0x02091C59C92fc5E0575b3B133c3caa4f57D79b0e;
+    address constant MCD_JOIN_RWA007_A              = 0xe3BA7B6c922cb4622AfF00C2e2A467fF6A06CA29;
+    address constant RWA007_A_URN                   = 0x78D9ccbe652E2aE7B11289B4F202278058aEfeeA;
+    address constant RWA007_A_JAR                   = 0xcF8A6445B6e22A8e2Fe9d7bd21410bB7a8b8D054;
+    // Goerli: Coinbase / Mainnet: Coinbase
+    address constant RWA007_A_OUTPUT_CONDUIT        = 0x9B4Af5F55b23cC2D86f58d7c08cD025bA1901013;
+    // Jar and URN Input Conduits
+    address constant RWA007_A_INPUT_CONDUIT_URN     = 0xFC80d4037dDadDb380e752AE46Aca262eEb812fC;
+    address constant RWA007_A_INPUT_CONDUIT_JAR     = 0x9ca5F7b1F5DCC287657c20547176a6733EB2c046;
+
+    // MIP21_LIQUIDATION_ORACLE params
+    string  constant RWA007_DOC                     = "QmRe77P2JsvQWygVr9ZAMs4SHnjUQXz6uawdSboAaj2ryF"; // TODO
+    // There is no DssExecLib helper, so WAD precision is used.
+    uint256 constant RWA007_A_INITIAL_PRICE         = 250_000_000 * WAD;
```

Comment:
just glancing at it but I think It's missing the script with the computation and the computed initial price, following previous patterns (`PE-1039`)

Reply:
Do you mean this?

https://github.com/makerdao/spells-goerli/pull/95/files#diff-6f5bde1acc013d43a68a5cce613f1afa7b0284903f20194c0b9b4cb31c9167ebR89-R96

Reply:
yes, assuming non zero stability fees.
double checking, it follow RWA009 pattern on initial price, resolving for now.